bin/hack-sound-server.in

```diff
@@ -27,6 +27,7 @@ gi.require_version('Gst', '1.0')
 gi.require_version('GLib', '2.0')
 from gi.repository import Gst  # noqa
 from gi.repository import GLib  # noqa
+from setproctitle import setproctitle
```

Comment:
Hmm I'm a bit worried about adding this dependency just for changing the proccess title. I looked at `setproctitle` and looks fine for portability in other operating systems like Windows, macOS. But since Hack works only in Linux, I wonder if there is a specific way for flatpak/Linux.

Is also a C library so ARM should be tested as well.

Reply:
You're right, I've tried this solution on aarch64 and this doesn't work. I'll try to find another solution.

Reply:
> You're right, I've tried this solution on aarch64 and this doesn't work. I'll try to find another solution.

I've been testing this today with a RPi4 (aarch64) and it worked this time. I think that I used a bad clubhouse flatpak manifest yesterday so the `setproctitle` dep was not installed, but today I built the version on this PR and tested on a real aarch64 device and it worked, so I think this is safe to be merged, after the clubhouse dep inclusion:

https://github.com/endlessm/clubhouse/pull/1180

Reply:
OK, thanks for checking. I'm also asking [here](https://pyar.discourse.group/t/como-cambiarle-el-nombre-al-proceso-python3/449) in a Python community. Let's wait a little bit. Maybe we can just wrap the sound server in a C wrapper that calls the Python script?

Reply:
Here is a good blog about adding a simple C wrapper http://edtheron.me/blog/embed-python-3-interpreter-c-program . The important line is the call to [Py_SetProgramName](https://docs.python.org/3/c-api/init.html#c.Py_SetProgramName).